exchange.c

```diff
@@ -162,6 +162,7 @@ static int domain_exchange_once(int (*layoutfunc)(int p), ExchangePlan * plan)
     for(i = 0; i < NumPart; i++)
     {
         if(!(P[i].OnAnotherDomain && P[i].WillExport)) continue;
+        if(P[i].IsGarbage) continue;
```

Comment:
Why would we want to export an garbage? Maybe the planning routine shall be able to know garbage is not supposed to be exported.

Reply:
Yes - that is what this commit does, isn't it? It skips exporting any garbage particle, also in the planning routine.